core/ui-common/src/main/kotlin/com/wire/android/util/DateAndTimeParsers.kt

```diff
@@ -156,7 +157,33 @@ class DateAndTimeParsers private constructor() {
                 null
             }
 
-        fun cellFileDateTime(instant: Instant): String = cellFileDateTimeFormat.format(instant.toJavaInstant())
+        fun cellTimeFormat(instant: Instant): String {
+            val timeFormatter = java.text.DateFormat.getTimeInstance(
+                java.text.DateFormat.SHORT,
+                Locale.getDefault()
+            ).apply {
+                timeZone = java.util.TimeZone.getDefault()
+            }
+            return timeFormatter.format(Date.from(instant.toJavaInstant()))
+        }
+
+        fun cellDateFormat(instant: Instant, showYear: Boolean = false): String {
+            val pattern = if (showYear) "MMM dd, yyyy" else "MMM dd"
+
+            val formatter = java.text.SimpleDateFormat(pattern, Locale.getDefault()).apply {
```

Comment:
Can we replace `SimpleDateFormat` pls 🙏  ? the same can be achieved through `java/kotlin.time` apis.
We replaced some time ago simple date formatters and `java.text` formatters in general.

- https://github.com/wireapp/wire-android/pull/3171

There are some findings in the ticket about why they are poor performant and some benchmarks :D 
https://wearezeta.atlassian.net/browse/WPB-9934